### PR TITLE
Add Endringslogg backend

### PR DESCRIPTION
**Pull request by aslaksm**
Endringslogg backend.
Uses flyway for migrations, migrations should run when docker container runs.

Env variables sources in `Env.kt` and `build.gradle.kts`. Database env variables are named according to [nais spec](https://doc.nais.io/persistence/postgres/), with assumed prefix DB.

Also check out url in `DbConnector.kt (line 40). Assumption is that NAIS will expose the GCP Cloud SQL instance through localhost (as per the docs) and that connection is straightforward. Not sure if this is actually the case, though.

### CODE REVIEW

File: endringslogg/src/main/kotlin/no/bekk/Application.kt [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        json(Json {
+            prettyPrint = true
+            encodeDefaults = true
+        })
+    }
+    install(CORS) {
+        // TODO: correctly define allowed hosts. Env. vars or secrets?
+        host("localhost:3000")
+        host("localhost:3333")
+        host("localhost:6006")
```
Comment (aslaksm) [inline endringslogg/src/main/kotlin/no/bekk/Application.kt:25]: Should be removed (or behind dev flag) before deploying)

File: endringslogg/src/main/kotlin/no/bekk/DbConnector.kt [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    val appId = varchar("app_id", 255)
+    val duration = integer("duration")
+    val unseenFields = integer("unseen_fields")
+    val timeStamp = timestamp("time_stamp")
+    override val primaryKey = PrimaryKey(userId, timeStamp)
+}
+
+fun connectToDatabase() {
+
+    val connectUrl: String = "jdbc:postgresql://$DB_HOST:$DB_PORT/$DB_DATABASE?reWriteBatchedInserts=true?sslmode=require"
```
Comment (aslaksm) [inline endringslogg/src/main/kotlin/no/bekk/DbConnector.kt:40]: Is this correct?

File: endringslogg/src/main/kotlin/no/bekk/env/Env.kt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+package no.bekk.env
+
+val DB_USERNAME: String = System.getenv("DB_USERNAME") ?: System.getenv("USER")
+val DB_PASSWORD = System.getenv("DB_PASSWORD") ?: ""
+val DB_HOST = System.getenv("DB_HOST") ?: "localhost"
+val DB_DATABASE = System.getenv("DB_DATABASE") ?: "endringslogg"
+val DB_PORT = System.getenv("DB_PORT") ?: "5432"
+val SANITY_PROJECT_ID: String = System.getenv("SANITY_PROJECT_ID") ?: "li581mqu"
```
Comment (aslaksm) [inline endringslogg/src/main/kotlin/no/bekk/env/Env.kt:8]: Assumption is made that NAV will only have one project. Alternative could be to pass this through frontend component.

**Review by stianStensli — APPROVED** [review]

File: endringslogg/build.gradle.kts [hunk 4]
```diff
@@ -0,0 +1,53 @@
+val ktor_version: String by project
+val kotlin_version: String by project
+val logback_version: String by project
+val exposed_version: String by project
+val DB_USERNAME: String = System.getenv("DB_USERNAME") ?: System.getenv("USER")
+val DB_PASSWORD = System.getenv("DB_PASSWORD") ?: ""
+val DB_HOST = System.getenv("DB_HOST") ?: "localhost"
+val DB_DATABASE = System.getenv("DB_DATABASE") ?: "endringslogg"
+val DB_PORT = System.getenv("DB_PORT") ?: "5432"
```
Comment (aslaksm) [inline endringslogg/build.gradle.kts:9]: Remove elvis

**Pull request merged**